src/helpers/urlHelper.ts

```diff
@@ -1,5 +1,5 @@
 const BASE_URL = process.env.REACT_APP_BASE_URL;
 
-export const getDownloadFileUrl = (fileId: string) => {
-  return `${BASE_URL}storage/${fileId}`;
+export const getDownloadFileUrl = (fileId: string | undefined) : string | null => {
+  return fileId ? `${BASE_URL}storage/${fileId}` : null;
```

Comment:
Now it looks like strange function :)
That was dumb formatter: get string -> return formatted string
Now added some strange logic (if you are not familiar with the project and this func usage, you probably determine it like some crutch). As for me looks like this logic should be somewhere else, where you can see it and immediately  tell why do we need such checks on undefined

Reply:
We might need to move it somewhere, I just tried to do as little as possible fixing the biggest issues :)

Reply:
Hmmm, ok, I don't know for now..
In vanilla js it looks ok for me to have something like `if (!filedId) return null;` at the start of the function
But in typescript it just looks weird because we have types and can define filedId exactly like string.. 
I hope you understand what I mean)

Reply:
![image](https://user-images.githubusercontent.com/10283222/210871884-8294733d-fdf8-490f-9c78-341b32fd138e.png)
idk, does it look better now?